docker/CMSRucioClient/scripts/CMSRSE.py

```diff
@@ -492,10 +504,30 @@ def _set_protocols(self):
             if new_proto['scheme'] in ['srm', 'srmv2', 'gsiftp', 'root', 'davs'] and not protocol_unchanged:
                 if self.dry:
                     logging.info('Adding %s to %s (Dry run, skipping)', new_proto['scheme'], self.rse_name)
+                    new_changes = True
                 else:
                     logging.info('Adding %s to %s', new_proto['scheme'], self.rse_name)
                     self.rcli.add_protocol(rse=self.rse_name, params=new_proto)
-        return
+                    new_changes = True
+
+        #delete protocols that are not in the new list
```

Comment:
We get the list of current protocols and match it with the one available in GitLab. 
If there are any extra protocols in our current list, we remove them.
( There are many SRM protocols definitions listed in many RSEs)